DeAceroDLabs/marketplace-frontend
# [DMP-463] Multiple forms not using all fields

## What is done

[[DMP-463] Multiple forms no tomaba en cuenta todos los fields](https://deacero.atlassian.net/browse/DMP-463)

## How is done

1. Add a `hidden` class on the Multiple Forms component to hide the forms that aren't being filled.
2. All the forms in the Multiple forms are rendered and always available from the beginning.
3. Previously, only the active form was rendered.

## How to test

1. Go to `/signup`.
2. Go to the last step.
3. Click on the new `finalizar` button.
4. An error should appear for the fields that are required.
5. Navigate to previous forms, the errors should be visible there too.
